packages/mds-compliance-service/ormconfig.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2019 City of Los Angeles
+ * Copyright 2021 City of Los Angeles
```

Comment:
All these `2019 => 2021` changes are not strictly necessary. Licenses are supposed to reflect the earliest date of publication.